nucypher-core/src/conditions.rs

```diff
@@ -0,0 +1,49 @@
+use alloc::string::String;
+
+use serde::{Deserialize, Serialize};
+
+/// Reencryption conditions.
+#[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
+pub struct Conditions(String);
+
+impl Conditions {
+    /// Creates a new conditions object.
```

Comment:
Is this comment likely to be helpful?

Reply:
I don't really have any better ideas. We could say here that it's supposed to be a JSON string, but nothing in `nucypher-core` really acts on that assumption...

Reply:
But do we need _any_ comment to explain that __new__ makes a new object?

Reply:
We do because of `#![warn(missing_docs)]`. It's easier to just write something than add a specific exclusion in this case.